packages/flutter_tools/templates/package/LICENSE.tmpl

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Your Company. All rights reserved.
```

Comment:
In that case we should also fix https://github.com/flutter/flutter/tree/master/packages/flutter_tools/templates/plugin